### PR TITLE
Add "Fast Bunny Hood" option

### PR DESCRIPTION
**Pull request by Kewlan**
Increases running speed by 50% while wearing the bunny hood.
Does _not_ increase animation speed. The patch might need to be applied somewhere else for that.

### CODE REVIEW

**Comment by gamestabled** [comment]
I want to wait on this and see if we can figure out the animation thing before merging it in, but still cool to see as is

File: code/Makefile [hunk 1]
```diff
@@ -54,7 +54,7 @@ endif
 
 VERFLAGS := -D OOT3D=$(OOT3D) -D OOT3DJ=$(OOT3DJ) -D Z3D=$(Z3D)
 
-ARCH	:=	-march=armv6k -mtune=mpcore -mfloat-abi=softfp -mtp=soft
+ARCH	:=	-march=armv6k -mtune=mpcore -mfloat-abi=softfp -mtp=soft -mfpu=vfpv2
```
Comment (lioncash) [inline code/Makefile]: Would you be able to split this out into its own PR in the meantime? This can be merged, given we should be specifying this anyway

Comment (Kewlan) [reply code/Makefile]: Done

Comment (lioncash) [reply code/Makefile]: Thanks!
